### PR TITLE
Crud for user

### PR DESCRIPTION
**Pull request by DavidGlusch**

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -42,10 +42,15 @@
...
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    'django.contrib.sites',
     "rest_framework",
     "book",
     "borrowing",
     "user",
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
```
Comment (yura-hudzovskyi) [inline config/settings.py:52]: fix quotes

File: config/settings.py [hunk 2]
```diff
@@ -131,3 +136,26 @@
...
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(days=2),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=3),
+    "ROTATE_REFRESH_TOKENS": False,
+}
+
+AUTHENTICATION_BACKENDS = [
+    'django.contrib.auth.backends.ModelBackend',
+    'allauth.account.auth_backends.AuthenticationBackend',
+]
```
Comment (yura-hudzovskyi) [inline config/settings.py:161]: add blank

File: config/settings.py [hunk 3]
```diff
@@ -42,10 +42,15 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    'django.contrib.sites',
```
Comment (oksana0410) [inline config/settings.py:45]: use ""

File: user/urls.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+app_name = "user"
+
+urlpatterns = [
+    path("register/", CreateUserView.as_view(), name="create"),
+    path("token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
+    path("token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
+    path("token/verify/", TokenVerifyView.as_view(), name="token_verify"),
+    path("me/", ManageUserView.as_view(), name="manage"),
+]
```
Comment (yura-hudzovskyi) [inline user/urls.py:18]: fix 
image

**Review by yura-hudzovskyi — CHANGES_REQUESTED** [review]

File: config/settings.py [hunk 5]
```diff
@@ -131,3 +136,26 @@
...
+}
+
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(days=2),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=3),
+    "ROTATE_REFRESH_TOKENS": False,
+}
+
+AUTHENTICATION_BACKENDS = [
+    'django.contrib.auth.backends.ModelBackend',
```
Comment (oksana0410) [inline config/settings.py:159]: fix quotes

**Pull request merged**